src/views/Dashboard/Hospital/Account/Certification.vue

```diff
@@ -169,34 +169,34 @@ export default {
     supportingDocumentsRules(){
       return [
         file => !file || file.type == 'application/pdf' || 'Document type should be application/pdf',
-        file => !file || file.size <= 2_000_000 || 'Document size should be less than 2 MB!',
+        file => !file || file.size <= 2_000_000 || 'The total file size uploaded exceeds the maximum file size allowed (2MB)',
       ]
     },
     titleRules() {
       return [
-        val => !!val || 'Title is required',
+        val => !!val || 'This field is required',
         val => (val && val.length <= 50) || 'Max 50 Character'
```

Comment:
Make the error message consistent with other field validation :
This field only allows 50 characters'